eZ/Publish/Core/Persistence/Cache/ContentTypeHandler.php

```diff
@@ -50,7 +51,13 @@ protected function init(): void
             ];
         };
 
-        $this->getTypeTags = static function (Type $type) { return ['type-' . $type->id]; };
+        $this->getTypeTags = static function (Type $type) {
+            return [
+                'type',
```

Comment:
Though I see a pattern here, it would be rather `type-list` here, based on other tags related to a list of objects.

Reply:
I briefly checked and see that `-list` suffix is used in keys rather than tags.

Reply:
> I briefly checked and see that `-list` suffix is used in keys rather than tags.

Hmm... ok, I'm not 100% convinced, but can't find similar case for tags indeed. So let's keep it for now.

Reply:
*Retro review*: This `type` tag was afaik not used anywhere, however I change the code to use it in order to get rid of `type-modifierId-` and `type-creatorId-` in #2840.

*Naming*: Given the use implies it's rather a "all types", name is ok.
*Lists*: List would be for a given list which could not be cleared by key, e.g. lots of variations by method signature.  however relevant use cases for that are slim, in most cases it's enough to just cache default arguments to be able to clear by key instead.